exp/views/responses.py

```diff
@@ -39,773 +50,626 @@
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__spouse_education_level"],
+        name="Parent education level",
+    ),
+    ResponseDataColumn(
+        id="demographic__annual_income",
+        description=(
+            "Parent's response to question 'What is your approximate family yearly income (in US dollars)?'; "
+            "options are 0, 5000, 10000, 15000, 20000-19000 in increments of 10000, >200000, or na [prefer not "
+            "to answer]"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__annual_income"],
+        name="Annual income",
+    ),
+    ResponseDataColumn(
+        id="demographic__number_of_books",
+        description="Parent's response to question 'About how many children's books are there in your home?'; integer",
+        extractor=lambda resp: resp["demographic_snapshot__number_of_books"],
+        name="Number of books",
+    ),
+    ResponseDataColumn(
+        id="demographic__additional_comments",
+        description="Parent's freeform response to question 'Anything else you'd like us to know?'",
+        extractor=lambda resp: resp["demographic_snapshot__additional_comments"],
+        name="Additional comments",
+    ),
+    ResponseDataColumn(
+        id="demographic__country",
+        description="Parent's response to question 'What country do you live in?'; 2-letter country code",
+        extractor=lambda resp: resp["demographic_snapshot__country"],
+        name="Country code",
+    ),
+    ResponseDataColumn(
+        id="demographic__state",
+        description=(
+            "Parent's response to question 'What state do you live in?' if country is US; 2-letter state "
+            "abbreviation"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__state"],
+        name="US State",
+    ),
+    ResponseDataColumn(
+        id="demographic__density",
+        description=(
+            "Parent's response to question 'How would you describe the area where you live?'; options are "
+            "urban, suburban, rural"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__density"],
+        name="Density",
+    ),
+    ResponseDataColumn(
+        id="demographic__lookit_referrer",
+        description="Parent's freeform response to question 'How did you hear about Lookit?'",
+        extractor=lambda resp: resp["demographic_snapshot__lookit_referrer"],
+        name="How you heard about Lookit",
+    ),
+]
 
-    headers_ordered = [name for (name, val, desc) in all_row_data][0:-2]
+# Which headers from the response data summary should go in the child data downloads
+CHILD_CSV_HEADERS = [
+    col.id
+    for col in RESPONSE_COLUMNS
+    if col.id.startswith("child__") or col.id.startswith("participant__")
+]
 
-    field_descriptions = {name: desc for (name, val, desc) in all_row_data}
+IDENTIFIABLE_DATA_HEADERS = [col.id for col in RESPONSE_COLUMNS if col.identifiable]
```

Comment:
Because I want to preserve the same order as in RESPONSE_COLUMNS. (And there are ~30 things in that list, and it basically shouldn't grow/change, so I wasn't concerned about speed.)

Reply:
ugh, sorry - brain-interpreter wasn't working 100% last night. Carry on!

Reply:
and to answer your question, no, in fact, iterating over sets is a bit slower:

```
❯ python -m timeit '[x for x in set(range(10000))]'
500 loops, best of 5: 655 usec per loop
❯ python -m timeit '[x for x in list(range(10000))]'
500 loops, best of 5: 504 usec per loop
```

Reply:
huh, interesting, thanks!